cms/envs/common.py

```diff
@@ -994,6 +994,9 @@
 XBLOCK_SETTINGS = {
     "VideoDescriptor": {
         "licensing_enabled": FEATURES.get("LICENSING", False)
+    },
+    'VideoModule': {
+        'YOUTUBE_API_KEY': YOUTUBE_API_KEY
```

Comment:
My strong preference is to have the ability to automate every phase of deployment.  So, I'd like to know what it would take to automate the generation of keys or whether that is even feasible.  Can you do so over the API?